docs/nidmm/functions.rst

```diff
@@ -462,7 +462,7 @@ nidmm.Session methods
 
 
         Specifies the Steinhart-Hart C coefficient for thermistor scaling when
-        Thermistor Type is set to Custom in the :py:func:`nidmm.configure_thermistor_type`
+        Thermistor Type is set to Custom in the :py:func:`nidmm.ConfigureThermistorType`
```

Comment:
How do we address this?

* Change source of extracted metadata to refer to the corresponding attribute. This would help us fix many, but not all of these problems.
* Just keep the C function name. It's really odd but better than a non-sensical name like we have now.
* Have metadata that tells our code-generator "if you encounter X, substitute with Y". This can get complex quickly and seems like a lot of work.

Reply:
Just like we override other information in the functions_addon, we could override the documentation. I don't like that idea because of the maintenance burden, but it would be a way to solve these issues.